src/components/AggregateCriteria/AggregateCriteriaCard.tsx

```diff
@@ -0,0 +1,129 @@
+import { Card, Col, Row, Select, TextField } from '@folio/stripes/components';
+import React, { useMemo } from 'react';
+import { CriteriaAggregateType } from '../../types/CriteriaTypes';
+import { Field, useField } from 'react-final-form';
+import OperatorSelect from '../Criteria/OperatorSelect';
+import useMonetaryOnBlur from '../../hooks/useMonetaryOnBlur';
+import { FormattedMessage, useIntl } from 'react-intl';
```

Comment:
just a suggestion to not mix 3rd party import with the rest, it's much easier to support such style